src/site/content/en/learn/css/css.11tydata.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function () {
+  return {
+    eleventyComputed: {
+      permalink: (data) => {
+        if (process.env.ELEVENTY_ENV === 'prod') {
```

Comment:
I feel like we want an `IGNORE_...` set of flags like as we do for d.c.c.. But this is fine if you want.

Reply:
Oh I think this'll be for a different purpose than what we do on d.c.c.

Kayce asked if I could add Andy to repo so he could work on the course. I wanted to make it so he could run the site locally and test the course, but nothing would go live until we remove this file.

Reply:
thanks for the review!